manifests/init.pp

```diff
@@ -38,15 +38,16 @@
 class virtualbox (
   String $version               = $virtualbox::params::version,
   String $package_ensure        = $virtualbox::params::package_ensure,
+  String $package_name          = $virtualbox::params::package_name,
```

Comment:
for which purpose do we add a parameter here?

Reply:
ah just reordered. Can you undo that?

Reply:
The package have not the same same on all OSes.
Some are `virtualbox` and others `Virtualbox`. This parameter was in master branch and works well.

Reply:
Ah in fact i used the order of the original puppet4 branch.
I reorder to stick with original master.

Reply:
In fact i remember why i follwoed the order in the puppet4 branch. All parameter related to package are grouped. And for me it is better to have all parameters named `package_` grouped together.